de.dlr.sc.virsat.model.extension.fdir.ui/src/de/dlr/sc/virsat/model/extension/fdir/ui/diagram/comments/CommentMoveFeature.java

```diff
@@ -0,0 +1,67 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.virsat.model.extension.fdir.ui.diagram.comments;
+
+import org.eclipse.graphiti.datatypes.ILocation;
+import org.eclipse.graphiti.features.IFeatureProvider;
+import org.eclipse.graphiti.features.context.IMoveShapeContext;
+import org.eclipse.graphiti.mm.pictograms.ContainerShape;
+import org.eclipse.graphiti.mm.pictograms.Diagram;
+import org.eclipse.graphiti.mm.pictograms.Shape;
+import org.eclipse.graphiti.services.Graphiti;
+
+import de.dlr.sc.virsat.graphiti.ui.diagram.feature.VirSatMoveShapeFeature;
+
+/**
+ * This class handles the drag & drop behavior of Fault Tree nodes for easy reconnection
+ * @author muel_s8
+ *
+ */
+public class CommentMoveFeature extends VirSatMoveShapeFeature {
+
+	/**
+	 * Standard constructor
+	 * @param fp the feature provider
+	 */
+	public CommentMoveFeature(IFeatureProvider fp) {
+		super(fp);
+	}
+
+	@Override
+	public boolean canMoveShape(IMoveShapeContext context) {
+		ContainerShape targetContainer = context.getTargetContainer();
```

Comment:
We need to take user rights management into account. 
If you dont own the SEI where the diagram is located in, then you shouldnt be allowed to perform a move.
I think the logic should be handled in the VirSatMoveShapeFeature.
So checking that the super method is true is well should deal with this.